src/authResolver.ts

```diff
@@ -163,6 +163,11 @@ export class RemoteSSHResolver implements vscode.RemoteAuthorityResolver, vscode
                         options = { shell: true, windowsHide: true, windowsVerbatimArguments: true };
                     }
 
+                    if (isFlatpak) {
+                        proxyArgs = ["--", "flatpak-spawn", "--host", "--env=TERM=xterm-256", proxyCommand, ...proxyArgs];
```

Comment:
Let me know if you would lime to use the `TERM` envvar defined locally if existing instead of forcing this xterm setup